sycl/cmake/modules/SYCLVersionFromVCS.cmake

```diff
@@ -0,0 +1,15 @@
+
+# Grab the date of the latest commit
+execute_process(
+  COMMAND git log -1 --format=%as # date in YYYY-MM-DD mode
+  WORKING_DIRECTORY ${SYCL_ROOT_SOURCE_DIR}
+  OUTPUT_VARIABLE GIT_COMMIT_DATE_TEMP
+  OUTPUT_STRIP_TRAILING_WHITESPACE
+)
+
+string(REPLACE "-" "" __LIBSYCL_TIMESTAMP ${GIT_COMMIT_DATE_TEMP})
```

Comment:
Please, remove this line and fix the git format instead.

Reply:
Done in f5418d7da2ef17ffebf5d53d0bdc193e522186a8